tensorflow/lite/micro/tools/make/gcc_download.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
```

Comment:
let's rename this to arm_gcc_download.sh

Reply:
ok